src/test/java/com/google/api/generator/engine/writer/JavaWriterVisitorTest.java

```diff
@@ -376,6 +377,53 @@ public void writeMethodInvocationExpr_chained() {
         writerVisitor.write(), "libraryClient.streamBooksCallable().doAnotherThing().call()");
   }
 
+  @Test
+  public void writeAnonymousClassExpr_basic() {
+    ConcreteReference ref = ConcreteReference.withClazz(Runnable.class);
+    TypeNode type = TypeNode.withReference(ref);
+    AnonymousClassExpr anonymousClassExpr = AnonymousClassExpr.builder().setType(type).build();
+    anonymousClassExpr.accept(writerVisitor);
+    assertEquals(writerVisitor.write(), "new Runnable() {\n}");
```

Comment:
Can we have this more closely resemble a real Runnable class implementation?

Reply:
yes, added the override run() method.